src/js/background/assignManager.js

```diff
@@ -571,6 +571,18 @@ window.assignManager = {
     return true;
   },
 
+  async _resetCookiesForSite(pageUrl, cookieStoreId) {
+    const url = new URL(pageUrl);
+    // Remove 'www.' from the domain value
+    const domain = url.hostname.replace(/^www\./, ""); 
+    const cookies = await browser.cookies.getAll({domain: domain, storeId: cookieStoreId});
+    for (const cookie of cookies) {
+      const domain = cookie.domain.startsWith(".") ? cookie.domain.slice(1) : cookie.domain;
+      const cookieUrl = `${cookie.secure ? "https" : "http"}://${domain}${cookie.path}`;
+      await browser.cookies.remove({ url: cookieUrl, name: cookie.name, storeId: cookie.storeId });   
```

Comment:
question (blocking): This code uses the `browser.cookies` API while the code in @dannycolin's PR uses the `browser.browsingData` API. Is there a strong reason to use `cookies` instead of `browsingData` or vice-versa?

Reply:
I changed it to use the browsingData API now! Thanks.